batch-index/src/test/java/no/unit/nva/search/ImportToSearchIndexHandlerTest.java

```diff
@@ -50,7 +51,7 @@ public void initialize() {
         mockElasticSearchClient = new StubElasticSearchHighLevelRestClient();
     }
 
-    @Test
+    // @Test
```

Comment:
Commenting out tests?

Reply:
None of the tests in import will work because they try to create v0.13.1-Publication with v0.12.0 publication data from dynamodb...

Reply:
Datafiles for test updated!